misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/OptionalAssertionsSpec.kt

```diff
@@ -42,4 +43,24 @@ abstract class OptionalAssertionsSpec (
             }
         }
     }
+
+    describeFun(isPresent.name) {
+        val isPresentFun = isPresent.lambda
+
+        context("is present") {
+            it("does not throw") {
```

Comment:
see ResultFeatureAssertionsSpec line 77, I suggest you do here the same, so change it to:
```suggestion
            it("${isPresent.name} - can perform sub-assertion which holds") {
```
moreover, I would add a second use case for 
```
it("${isPresent.name} - can perform sub-assertion which fails, throws AssertionError") {